modules/sdk-test-multi-cs/sdk-test-multi-cs.csproj

```diff
@@ -1,6 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <TargetName>StdbModule</TargetName>
+    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
```

Comment:
Let's remove these? They're useful for local debugging, but not necessary for e.g. CI. For checking the output, we have Codegen.Tests anyway these days.

Reply:
Ah yeah the second one is a mistake, but I'd like to keep the former; for the simple reason it allows all test modules to be loaded in the sln found under bindings-csharp; otherwise they conflict on the same project name.

For wasm StdbModule is hardcoded, so TargetName fixes that.

Reply:
That's fine by me.